tests/test87.c

```diff
@@ -14,7 +14,7 @@ hstruct_t tst[] = {{{}, "muh1", 2}, {{}, "muh2", 8}, {{}, "muh3", 1}, {{}, "muh4
   {{}, "muh7", 6}, {{}, "muh8", 15}, {{}, "muh9", 6}, {{}, "muh10", 9}, {{}, "muh11", 10}, {{}, "muh12", 43}};
 hstruct_t *hTable;
 
-static int cmpfunc(hstruct_t *s1, hstruct_t *s2)
+static int cmpfunc(const hstruct_t *s1, const hstruct_t *s2)
```

Comment:
https://github.com/troydhanson/uthash/commit/efd5ea2ed47997a8d2fbe108f18a4a4a619622ee#diff-d204ed043584be02f656985687fbc6e0 — thanks!  (I ran `astyle` while I was at it.)